pkg/cmd/pr/comment/comment.go

```diff
@@ -76,9 +76,9 @@ func NewCmdComment(f *cmdutil.Factory, runF func(*shared.CommentableOptions) err
 	cmd.Flags().StringVarP(&bodyFile, "body-file", "F", "", "Read body text from `file` (use \"-\" to read from standard input)")
 	cmd.Flags().BoolP("editor", "e", false, "Skip prompts and open the text editor to write the body in")
 	cmd.Flags().BoolP("web", "w", false, "Open the web browser to write the comment")
-	cmd.Flags().BoolVar(&opts.EditLast, "edit-last", false, "Edit the last comment of the same author")
-	cmd.Flags().BoolVar(&opts.DeleteLast, "delete-last", false, "Delete the last comment of the same author")
-	cmd.Flags().BoolVar(&opts.Confirmed, "yes", false, "Skip the confirmation prompt")
+	cmd.Flags().BoolVar(&opts.EditLast, "edit-last", false, "Edit the last comment of the current user")
+	cmd.Flags().BoolVar(&opts.DeleteLast, "delete-last", false, "Delete the last comment of the current user")
+	cmd.Flags().BoolVar(&opts.DeleteLast, "yes", false, "Skip the delete confirmation prompt when --delete-last is provided")
```

Comment:
This should be `opts.DeleteLastConfirmed`.

Reply:
thanks for the catch